public/utils/contextMenu/getActions.tsx

```diff
@@ -36,7 +41,6 @@ export const getActions = ({ core, plugins }) => {
                 {...{
```

Comment:
This spread operator can be removed. In other words, just have `{ <content> }` instead of `{ ... { <content> } }`